Examples/Algorithms/Generators/ActsExamples/Generators/ParametricParticleGenerator.hpp

```diff
@@ -35,8 +35,14 @@ class ParametricParticleGenerator : public EventGenerator::ParticlesGenerator {
     ///
     /// This intentionally uses theta instead of eta so it can represent the
     /// full direction space with finite values.
+    ///
+    /// @note This is the standard generation, for detector performance
+    /// classification, where a flat distribution in eta can be useful,
+    /// this can be set by the forceEtaGeneration flag;
+    ///
     double thetaMin = 0.0;
     double thetaMax = M_PI;
+    bool forceEtaGeneration = false;
```

Comment:
Can we also call this `sampleEta` or `uniformEta`?

Reply:
Absolutely, will change.

Reply:
changed.